deployer/src/test/java/eu/seaclouds/location/cloudfoundry/PhpWebCloudFoundryLiveTest.java

```diff
@@ -39,9 +39,7 @@
 
 public class PhpWebCloudFoundryLiveTest extends AbstractCloudFoundryPaasLocationLiveTest {
 
-    private final String APPLICATION_URL = checkNotNull(getClass().getClassLoader()
-            .getResource("phpHelloWorld")).getFile();
-
+    private final String APPLICATION_URL = "https://github.com/kiuby88/phpHelloWorld/releases/download/1/phpHelloWorld.zip";
```

Comment:
Is there any reason for not having the zip file in some test resources folder inside https://github.com/SeaCloudsEU/SeaCloudsPlatform/....  ??